translate country on monument page (bug 39865)

jdlrobson opened the pull request.

jrobson committed:
translate country on monument page (bug 39865)

bvibber commented:
I don't see a translated country on the monument detail page; I see no campaign information at all...

jrobson committed:
order matters (caching)

ensure that the element with class campaign is ready before doing
translation
(see https://github.com/wikimedia/WLMMobile/pull/241#issuecomment-8285251)

jdlrobson commented:
fixed :)
